runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordPlugin.java

```diff
@@ -381,6 +381,19 @@ private void checkForAreaUpdate()
 			return;
 		}
 
+		final EnumSet<WorldType> worldType = client.getWorldType();
+
+		if (worldType.contains(WorldType.DEADMAN))
```

Comment:
this should return playing DMM even for tournament DMM worlds imo

Reply:
There was a comment in the code that tournament worlds were also flagged deadman.

Reply:
Nvm, I just read the comment in PVP_WORLD_TYPES, this is fine